### ISSUE TITLE
Don't nag triage owners about mismatched severity and accessibility severity for no-plan-to-ship bugs

### ISSUE THREAD

**Issue by jcsteh** [mozilla/bugbot#2435]
There are some access s2 bugs in features which are preffed off in all channels and we have no plans to ship them any time soon. On one hand, it's not shipping, so it's not currently an access s2 for users. On the other hand, if we later decide to ship it, it will become an access s2. We do not want to downgrade these to access s3s because when we decide to ship, folks might not realise that they are ship blockers.

To deal with this, we [now have a no-plan-to-ship keyword](https://bugzilla.mozilla.org/show_bug.cgi?id=1901188). BugBot should avoid asking triage owners about a mismatch between the severity and accessibility severity fields when this keyword is set. Once the keyword is unset, then the triage owner should be asked.

**Comment by marco-c** [comment]
It should be easy to implement this by changing the query in https://github.com/mozilla/bugbot/blob/master/bugbot/rules/severity_inconsistency.py.
